groups/chd-infrastructure/main.tf

```diff
@@ -0,0 +1,31 @@
+# ------------------------------------------------------------------------------
+# Providers
+# ------------------------------------------------------------------------------
+terraform {
+  required_version = ">= 0.13.0, < 0.14"
+
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = ">= 0.3, < 4.0"
+    }
+    vault = {
+      source  = "hashicorp/vault"
+      version = ">= 2.0.0"
+    }
+  }
+  backend "s3" {}
+}
+
+provider "aws" {
+  region = var.aws_region
+}
+
+provider "vault" {
```

Comment:
I would pull this out as a separate file as done [elsewhere](https://github.com/companieshouse/logging-stack/blob/master/groups/elasticsearch/vault.tf) so it'll be easier to migrate to a role later on

Reply:
Updated in commit 9f8899a